site/src/components/Markdown/Markdown.tsx

```diff
@@ -0,0 +1,93 @@
+import Link from "@material-ui/core/Link"
+import { Theme, useTheme } from "@material-ui/core/styles"
+import Table from "@material-ui/core/Table"
+import TableBody from "@material-ui/core/TableBody"
+import TableCell from "@material-ui/core/TableCell"
+import TableContainer from "@material-ui/core/TableContainer"
+import TableHead from "@material-ui/core/TableHead"
+import TableRow from "@material-ui/core/TableRow"
+import { FC } from "react"
+import ReactMarkdown from "react-markdown"
+import SyntaxHighlighter from "react-syntax-highlighter"
+import { dracula as dark } from "react-syntax-highlighter/dist/esm/styles/hljs"
+import gfm from "remark-gfm"
+
+export interface MarkdownProps {
+  children: string
+}
+
+export const Markdown: FC<{ children: string }> = ({ children }) => {
+  const theme: Theme = useTheme()
+
+  return (
+    <ReactMarkdown
+      remarkPlugins={[gfm]}
+      components={{
+        a: ({ href, target, children }) => (
+          <Link href={href} target={target}>
+            {children}
+          </Link>
+        ),
+
+        // Adding node so the ...props don't have it
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        code: ({ node, inline, className, children, ...props }) => {
+          const match = /language-(\w+)/.exec(className || "")
+          return !inline && match ? (
+            <SyntaxHighlighter
+              // Custom style to match our main colors
+              style={{
+                ...dark,
+                hljs: {
+                  ...dark.hljs,
+                  background: theme.palette.background.default,
+                  borderRadius: theme.shape.borderRadius,
+                  color: theme.palette.text.primary,
+                },
+              }}
+              language={match[1]}
+              PreTag="div"
+              {...props}
+            >
+              {String(children).replace(/\n$/, "")}
```

Comment:
How come we have to do this `.replace`? Is it to remove line breaks or something?

Reply:
Yes, it is so the highlight component can struct it properly. I got this from the `react-markdown` docs https://github.com/remarkjs/react-markdown#use-custom-components-syntax-highlight